benchmarks/h3.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+const { createServer } = require('http')
+const { createApp } = require('h3')
+
+const app = createApp()
+
+app.useAsync('/', () => ({ hello: 'world' }))
```

Comment:
The router is fast skipped for this case. Is it still eligible to set `hasRouter` flag or are there any baselines a router should handle?

Reply:
cc: @mcollina